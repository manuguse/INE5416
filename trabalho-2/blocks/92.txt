1 1 6 6 11 16 16 16
2 2 6 11 11 17 17 16
3 6 6 11 11 11 16 16
4 7 6 12 12 18 18 18
4 7 10 10 14 14 18 18
4 4 4 13 15 15 19 19
4 8 9 13 13 19 19 19
5 9 9 9 13 13 13 13