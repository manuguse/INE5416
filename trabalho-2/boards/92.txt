- - 5 - - 4 1 6
- - - 5 4 - - -
- - 3 - - 2 - -
6 - 1 - - - 5 4
- - 2 - - - 2 -
- 4 - - - - - 4
2 - 3 - 6 1 2 -
- 4 - - - 3 4 7