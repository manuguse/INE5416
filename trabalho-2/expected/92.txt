1 2 5 2 6 4 1 6
2 1 4 5 4 1 2 5
1 6 3 1 3 2 3 2
6 2 1 2 1 3 5 4
5 1 2 1 2 1 2 1
3 4 1 2 1 2 5 4
2 1 3 1 6 1 2 3
1 4 1 2 5 3 4 7